### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by sedited**
Translated the remaining fields to German.

### CODE REVIEW

**Comment by Dikay900** [comment]
This includes the PR #79 
you can squash commit 063dc96 from this PR

File: src/qt/locale/bitcoin_de.ts [hunk 1]
```diff
@@ -262,7 +268,7 @@ Dieses Produkt enthält Software, die vom OpenSSL-Projekt zur Verwendung im Open
     <message>
         <location line="-58"/>
         <source>Gridcoin will close now to finish the encryption process. Remember that encrypting your wallet cannot fully protect your coins from being stolen by malware infecting your computer.</source>
-        <translation type="unfinished"/>
+        <translation>Gridcoin wird jetzt runterfahren um den Verschlüsselungsprozess zu vervollständigen. Errinnern Sie sich, dass das Verschlüsseln ihres Wallets nicht reicht um es vollständig vor Diebstahl zu schützen, z.B. durch. Viren, die ihren Rechner infiziert haben.</translation>
```
Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts]: typo 'Erinnern'

File: src/qt/locale/bitcoin_de.ts [hunk 2]
```diff
@@ -39,7 +43,7 @@ Dieses Produkt enthält Software, die vom OpenSSL-Projekt zur Verwendung im Open
     <message>
         <location filename="../forms/addressbookpage.ui" line="+14"/>
         <source>Address Book</source>
-        <translation>Adressbuch</translation>
+        <translation>Eigene Adresse</translation>
```
Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts]: Are these really your own adresses there? Address Book should be target adresses right?

Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts]: Could be more than one so maybe "Eigene Adressen" ? BTW are these our own addresses? Shouldn't this be under "Receive Coins" ?

File: src/qt/locale/bitcoin_de.ts [hunk 3]
```diff
@@ -538,67 +547,82 @@ Dieses Produkt enthält Software, die vom OpenSSL-Projekt zur Verwendung im Open
...
+        <translation>
+            <numerusform>%n aktive Verbindung(en) zum Gridcoin Netzwerk</numerusform>
+            <numerusform></numerusform>
+        </translation>
     </message>
     <message>
         <location line="+40"/>
         <source>Downloaded %1 blocks of transaction history.</source>
-        <translation type="unfinished"/>
+        <translation>%1 Blöcke der Transaktiongeschichte heruntegeladen.</translation>
```
Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts]: typo 'heruntergeladen'

File: src/qt/locale/bitcoin_de.ts [hunk 4]
```diff
@@ -613,17 +637,17 @@ Dieses Produkt enthält Software, die vom OpenSSL-Projekt zur Verwendung im Open
...
         <location line="+10"/>
         <source>Last received block was generated %1.</source>
-        <translation type="unfinished"/>
+        <translation>Letzter erhaltener Block wurde vor %1 generiert.</translation>
     </message>
     <message>
         <location line="+59"/>
         <source>This transaction is over the size limit.  You can still send it for a fee of %1, which goes to the nodes that process your transaction and helps to support the network.  Do you want to pay the fee?</source>
-        <translation type="unfinished"/>
+        <translation>Diese Transaktion ist über der Grössenlimite. Sie können sie immer noch für eine Gebühr über %1 verschicken, die an die Knoten geht, die ihre Transaktion bearbeiten und unterstützt damit das Netzwerk. Wollen sie diese Gebühr bezahlen?</translation>
```
Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts]: better maybe 'über dem Grössenlimit [...} und unterstützen damit das Netzwerk.'

File: src/qt/locale/bitcoin_de.ts [hunk 5]
```diff
@@ -613,17 +637,17 @@ Dieses Produkt enthält Software, die vom OpenSSL-Projekt zur Verwendung im Open
...
     </message>
     <message>
         <location line="+5"/>
         <source>Confirm transaction fee</source>
-        <translation type="unfinished"/>
+        <translation>Bestätige Transaktiongebühren</translation>
```
Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts]: typo 'Transaktionsgebühren'

File: src/qt/locale/bitcoin_de.ts [hunk 6]
```diff
@@ -651,13 +675,13 @@ Adresse: %4</translation>
...
         <source>URI handling</source>
-        <translation type="unfinished"/>
+        <translation>URI Handhabung</translation>
     </message>
     <message>
         <location line="-15"/>
         <location line="+15"/>
         <source>URI can not be parsed! This can be caused by an invalid Gridcoin address or malformed URI parameters.</source>
-        <translation type="unfinished"/>
+        <translation>URI kann nicht geparsed werde ! Dies kann durch eine nicht gültige Gridcoin Addresse, oder schlechten URI Parametern verursacht worden sein.</translation>
```
Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts]: typo 'geparst werden'

File: src/qt/locale/bitcoin_de.ts [hunk 7]
```diff
@@ -672,52 +696,64 @@ Adresse: %4</translation>
...
+        <translation>
+            <numerusform>%n Tag</numerusform>
+            <numerusform>%n Tage</numerusform>
+        </translation>
     </message>
     <message>
         <location line="+18"/>
         <source>Not staking</source>
-        <translation type="unfinished"/>
+        <translation>Nicht am Veryinsen</translation>
```
Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts]: typo 'Verzinsen'

File: src/qt/locale/bitcoin_de.ts [hunk 8]
```diff
@@ -947,7 +987,11 @@ Adresse: %4</translation>
 This label turns red, if the priority is smaller than &quot;medium&quot;.
 
  This means a fee of at least %1 per kb is required.</source>
-        <translation type="unfinished"/>
+        <translation>Transaktionen mit einer höheren Priorität kommen wahrscheinlicher in einen Block.
+
+Diese Label wird rot, wenn die Priorität kleiner ist als Mittel.
+
+ Dies bedeutet, dass eine Gebühr von mindesten %1 pro kB benötigt wird.</translation>
```
Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts]: typo 'Dieses Label'
typo 'mindestens'

File: src/qt/locale/bitcoin_de.ts [hunk 9]
```diff
@@ -956,14 +1000,20 @@ This label turns red, if the priority is smaller than &quot;medium&quot;.
...
+
+ Beträge 0.546-mal die Minimalgebühr werden als DUST angezeigt.</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>This label turns red, if the change is smaller than %1.
 
  This means a fee of at least %2 is required.</source>
-        <translation type="unfinished"/>
+        <translation>Dieses Label wird rot, wenn die Veränderung kleiner als %1 ist.
```
Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts]: 'die Veränderung' -> 'das Wechselgeld'

File: src/qt/locale/bitcoin_de.ts [hunk 10]
```diff
@@ -956,14 +1000,20 @@ This label turns red, if the priority is smaller than &quot;medium&quot;.
...
+
+ Dies bedeutet, dass eine Gebühr von %2 benötigt wird.</translation>
```
Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts]: better 'von mindestens'

File: src/qt/locale/bitcoin_de.ts [hunk 11]
```diff
@@ -1109,7 +1159,7 @@ This label turns red, if the priority is smaller than &quot;medium&quot;.
     <message>
         <location line="+6"/>
         <source>Optional transaction fee per kB that helps make sure your transactions are processed quickly. Most transactions are 1 kB. Fee 0.01 recommended.</source>
-        <translation type="unfinished"/>
+        <translation>Optional Transaktionsgebühr pro Kilobyte, die hilft Transaktionen schneller zu bearbeiten. Die Meisten Transaktionen sind 1kB. Di Gebühr 0.01GRC wird vorgeschlagen.</translation>
```
Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts]: typo 'Die meisten'
typo 'Die Gebühr'

File: src/qt/locale/bitcoin_de.ts [hunk 12]
```diff
@@ -1139,12 +1189,12 @@ This label turns red, if the priority is smaller than &quot;medium&quot;.
     <message>
         <location line="+7"/>
         <source>Detach block and address databases at shutdown. This means they can be moved to another data directory, but it slows down shutdown. The wallet is always detached.</source>
-        <translation type="unfinished"/>
+        <translation>Trenne Datenbank beim Runterhafen. Dies bedeutet, dass sie in ein anderes Verzeichnis verlegt werden können, jedoch verzörgert verlängert dies die benötigte Zeit zum Runterfahren. Das Wallet ist immer getrennt.</translation>
```
Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts]: 'verzörgert ' can be dropped here

File: src/qt/locale/bitcoin_de.ts [hunk 13]
```diff
@@ -1325,7 +1375,7 @@ This label turns red, if the priority is smaller than &quot;medium&quot;.
     <message>
         <location line="-160"/>
         <source>Stake:</source>
-        <translation type="unfinished"/>
+        <translation>Einsatz:</translation>
```
Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts:1378]: maybe better and consistent 'Verzinst:'

File: src/qt/locale/bitcoin_de.ts [hunk 14]
```diff
@@ -1690,12 +1740,12 @@ This label turns red, if the priority is smaller than &quot;medium&quot;.
     <message>
         <location line="+35"/>
         <source>Change</source>
-        <translation type="unfinished"/>
+        <translation>Veränderung</translation>
```
Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts]: better/consistent 'Wechselgeld'

File: src/qt/locale/bitcoin_de.ts [hunk 15]
```diff
@@ -2679,12 +2741,12 @@ This label turns red, if the priority is smaller than &quot;medium&quot;.
     <message>
         <location line="+5"/>
         <source>Bind to given address. Use [host]:port notation for IPv6</source>
-        <translation type="unfinished"/>
+        <translation>Binde an gegebene Addresse. Benutze [host]:port Notaion für IPv6</translation>
```
Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts]: typo 'Notation'

File: src/qt/locale/bitcoin_de.ts [hunk 16]
```diff
@@ -2704,7 +2766,7 @@ This label turns red, if the priority is smaller than &quot;medium&quot;.
     <message>
         <location line="+51"/>
         <source>Detach block and address databases. Increases shutdown time (default: 0)</source>
-        <translation type="unfinished"/>
+        <translation>Löse Block und Addresse Datenbank ab. Ehöhe die Zeit beim Runterfahren (default: 0)</translation>
```
Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts]: 'Erhöht'

File: src/qt/locale/bitcoin_de.ts [hunk 17]
```diff
@@ -2769,12 +2831,12 @@ This label turns red, if the priority is smaller than &quot;medium&quot;.
     <message>
         <location line="+117"/>
         <source>Error initializing database environment %s! To recover, BACKUP THAT DIRECTORY, then remove everything from it except for wallet.dat.</source>
-        <translation type="unfinished"/>
+        <translation>Fehler beim initiallisieren der Datenbank Umgebung %s! Um wiederherzustellen BACKUP DAS VERZEICHNIS, dann entferne alles davon ausser die wallet.dat Datei. </translation>
```
Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts]: typo 'initialisieren'

File: src/qt/locale/bitcoin_de.ts [hunk 18]
```diff
@@ -2769,12 +2831,12 @@ This label turns red, if the priority is smaller than &quot;medium&quot;.
...
     </message>
     <message>
         <location line="-20"/>
         <source>Set maximum size of high-priority/low-fee transactions in bytes (default: 27000)</source>
-        <translation type="unfinished"/>
+        <translation>Set maximalgrösse von hoch/tief Gebühren Transaktionen in Bytes (default:27000)</translation>
```
Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts]: better 'Setze Maximalgrösse von hoch priorisierte Transaktionen mit niedrigen Gebühren in [...]'

File: src/qt/locale/bitcoin_de.ts [hunk 19]
```diff
@@ -2824,22 +2886,22 @@ This label turns red, if the priority is smaller than &quot;medium&quot;.
...
         <location line="-90"/>
         <source>Find peers using DNS lookup (default: 1)</source>
-        <translation type="unfinished"/>
+        <translation>Finde Peers mit DNS lookup (default:1)</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Sync checkpoints policy (default: strict)</source>
-        <translation type="unfinished"/>
+        <translation>Synchronisiere Checkpoints Police (default:streng)</translation>
```
Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts]: default should still be 'strict' i think

File: src/qt/locale/bitcoin_de.ts [hunk 20]
```diff
@@ -2859,17 +2921,17 @@ This label turns red, if the priority is smaller than &quot;medium&quot;.
     <message>
         <location line="+28"/>
         <source>Output extra debugging information. Implies all other -debug* options</source>
-        <translation type="unfinished"/>
+        <translation>Ausgabe weiterer Debug Informationen. Implziert alle anderen -debug* Optionen</translation>
```
Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts]: typo 'Impliziert'

File: src/qt/locale/bitcoin_de.ts [hunk 21]
```diff
@@ -2859,17 +2921,17 @@ This label turns red, if the priority is smaller than &quot;medium&quot;.
...
         <location line="+1"/>
         <source>Output extra network debugging information</source>
-        <translation type="unfinished"/>
+        <translation>Ausgabe weiterer Netzwerk Debugging Informationen</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Prepend debug output with timestamp</source>
-        <translation type="unfinished"/>
+        <translation>Füge Debug Ausgabe mit Timestamp an</translation>
```
Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts]: Timestamp -> 'Zeitstempel'
better 'Füge Zeitstempel zu Debug Ausgabe hinzu'

File: src/qt/locale/bitcoin_de.ts [hunk 22]
```diff
@@ -2879,7 +2941,7 @@ This label turns red, if the priority is smaller than &quot;medium&quot;.
     <message>
         <location line="-74"/>
         <source>Select the version of socks proxy to use (4-5, default: 5)</source>
-        <translation type="unfinished"/>
+        <translation>Wähle die Socks Proxz Version (4-5,default:5)</translation>
```
Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts]: typo 'Proxy'

File: src/qt/locale/bitcoin_de.ts [hunk 23]
```diff
@@ -2915,7 +2977,7 @@ This label turns red, if the priority is smaller than &quot;medium&quot;.
         <location line="+109"/>
         <source>Unable to sign checkpoint, wrong checkpointkey?
 </source>
-        <translation type="unfinished"/>
+        <translation>Ausser Stange den checkpoint zu unterschreiben - falscher checkpointkey?</translation>
```
Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts]: better 'Nicht möglich den Checkpoint [...]'

File: src/qt/locale/bitcoin_de.ts [hunk 24]
```diff
@@ -3080,12 +3151,12 @@ for example: alertnotify=echo %%s | mail -s &quot;Gridcoin Alert&quot; admin@foo
     <message>
         <location line="+1"/>
         <source>Acceptable ciphers (default: TLSv1+HIGH:!SSLv2:!aNULL:!eNULL:!AH:!3DES:@STRENGTH)</source>
-        <translation type="unfinished"/>
+        <translation>Annembahre Chiffren (default: TLSv1+HIGH:!SSLv2:!aNULL:!eNULL:!AH:!3DES:@STRENGTH)</translation>
```
Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts]: typo 'Annehmbare'

File: src/qt/locale/bitcoin_de.ts [hunk 25]
```diff
@@ -3080,12 +3151,12 @@ for example: alertnotify=echo %%s | mail -s &quot;Gridcoin Alert&quot; admin@foo
...
     </message>
     <message>
         <location line="+53"/>
         <source>Error: Wallet unlocked for staking only, unable to create transaction.</source>
-        <translation type="unfinished"/>
+        <translation>Fehler: Das Wall ist nur zum Verzinsen geöffnet, Ausser Stance die Transaktion zu erstellen.</translation>
```
Comment (Dikay900) [inline src/qt/locale/bitcoin_de.ts]: typo 'Wallet'
better 'Ausser Stance' -> 'nicht möglich'

**Comment by Dikay900** [comment]
reviewed, would be great if you can see my comments. This is a great addition to the german translation. good job

**Comment by sedited** [comment]
Thanks for you time! Corrected the typos and more. Grcnation uses the terms "Stake" and "Zinsen", which I tried to follow, but probably misinterpreted in some instances. Maybe a Dictionary should be agreed upon, since the technical jargon can be very confusing. The wallet for example has been translated to three different terms: "Wallet", "Brieftasche", "Client"...

**Comment by Dikay900** [comment]
yeah the wallet thing came to my mind too but I personally do not think that this is a big impact in understanding the GUI.
maybe this can be fixxed in a followup PR

looks good to me can be merged :+1: 
@gridcoin

**Comment by gridcoin** [comment]
Thanks.

**Pull request merged**